test/e2e/set/scyllacluster/scyllacluster_external_seeds.go

```diff
@@ -0,0 +1,80 @@
+	"github.com/scylladb/scylla-operator/test/e2e/framework"
+	"github.com/scylladb/scylla-operator/test/e2e/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = g.Describe("MultiDC cluster", func() {
+	defer g.GinkgoRecover()
+
+	f := framework.NewFramework("scyllacluster")
+
+	g.It("should form when external seeds are provided to ScyllaClusters", func() {
+		ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
+		defer cancel()
+
+		sc1 := scyllafixture.BasicScyllaCluster.ReadOrFail()
+		sc1.Name = "basic-cluster"
+		sc1.Spec.Datacenter.Name = "us-east-1"
+		sc1.Spec.Datacenter.Racks[0].Name = "us-east-1a"
+		sc1.Spec.Datacenter.Racks[0].Members = 2
+
+		framework.By("Creating first ScyllaCluster")
+		sc1, err := f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Create(ctx, sc1, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the first ScyllaCluster to rollout (RV=%s)", sc1.ResourceVersion)
+		waitCtx1, waitCtx1Cancel := utils.ContextForRollout(ctx, sc1)
+		defer waitCtx1Cancel()
+		sc1, err = utils.WaitForScyllaClusterState(waitCtx1, f.ScyllaClient().ScyllaV1(), sc1.Namespace, sc1.Name, utils.WaitForStateOptions{}, utils.IsScyllaClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		verifyScyllaCluster(ctx, f.KubeClient(), sc1)
+		hosts1 := getScyllaHostsAndWaitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc1)
+		di1 := insertAndVerifyCQLData(ctx, hosts1)
+		defer di1.Close()
+
+		ns2 := f.CreateAndSetupSecondaryNamespace(ctx)
+
+		sc2 := scyllafixture.BasicScyllaCluster.ReadOrFail()
+		sc2.Name = "basic-cluster"
+		sc2.Spec.Datacenter.Name = "us-east-2"
+		sc2.Spec.Datacenter.Racks[0].Name = "us-east-2a"
+		sc2.Spec.Datacenter.Racks[0].Members = 2
+		sc2.Spec.ExternalSeeds = []string{
+			naming.CrossNamespaceServiceNameForCluster(sc1),
+		}
+
+		framework.By("Creating second ScyllaCluster")
+		sc2, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(ns2.Name).Create(ctx, sc2, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the second ScyllaCluster to rollout (RV=%s)", sc2.ResourceVersion)
+		waitCtx2, waitCtx2Cancel := utils.ContextForRollout(ctx, sc2)
+		defer waitCtx2Cancel()
+		sc2, err = utils.WaitForScyllaClusterState(waitCtx2, f.ScyllaClient().ScyllaV1(), sc2.Namespace, sc2.Name, utils.WaitForStateOptions{}, utils.IsScyllaClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		verifyScyllaCluster(ctx, f.KubeClient(), sc2)
+
+		framework.By("Verifying a multi datacenter cluster was formed with the first ScyllaCluster")
+		hostsByDC := getScyllaHostsByDCAndWaitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc1, sc2)
+		o.Expect(hostsByDC[sc1.Spec.Datacenter.Name]).To(o.ConsistOf(hosts1))
+
+		di2 := insertAndVerifyCQLDataByDC(ctx, hostsByDC)
+		defer di2.Close()
+
+		framework.By("Verifying data of datacenter %q", sc1.Spec.Datacenter.Name)
+		verifyCQLData(ctx, di1)
```

Comment:
validate whether cluster status pulled from scyllaclient consist of all expected hosts and their datacenter information matches names from both SCs

Reply:
> validate whether cluster status pulled from scyllaclient consist of all expected hosts

this is already done in `getScyllaHostsByDCAndWaitForFullQuorum`

> their datacenter information matches names from both SCs

added